src/catalog/src/durable/persist.rs

```diff
@@ -334,6 +337,15 @@ impl PersistHandle {
         }
         let current_epoch = Epoch::new(current_epoch).expect("known to be non-zero");
 
+        debug!(
+            "open_inner is_initialized={:?} upper={:?} user_version={:?} prev_epoch={:?} current_epoch={:?}",
+            is_initialized,
+            upper,
+            user_version,
+            prev_epoch,
+            current_epoch,
+        );
```

Comment:
Just an FYI, it turns out you can write the trace like 
```
debug!(?is_initialized,?upper, ?user_version, ?prev_epoch, ?current_epoch, "open inner")
```
and it will trace the log lines with the variable names, exactly like you have here. Doesn't matter to me which you chose, I just found this other way to be less typing

Reply:
Oh awesome, that looks much cleaner, I'll switch to that.